xla/service/gpu/transforms/cudnn_sdpa_score_mod_test.cc

```diff
@@ -0,0 +1,135 @@
+/* Copyright 2025 The OpenXLA Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+#include "xla/service/gpu/transforms/cudnn_sdpa_score_mod.h"
+
+#include <gmock/gmock.h>
+#include <gtest/gtest.h>
+
+#include "absl/status/status.h"
+#include "json/json.h"
+#include "third_party/cudnn_frontend/include/cudnn_frontend.h"
+#include "tsl/platform/status_matchers.h"
+#include "tsl/platform/test.h"
+#include "xla/hlo/parser/hlo_parser.h"
+
+namespace stream_executor {
+namespace gpu {
+
+using tsl::testing::IsOk;
+
+TEST(CudnnSdpaScoreModTest, CompileFwd) {
+  absl::string_view hlo = R"(
+  HloModule jit__unnamed_wrapped_function_
+
+  %fwd (Arg_0.15: f32[4,4,1024,1024]) -> f32[4,4,1024,1024] {
+    %Arg_0.15 = f32[4,4,1024,1024]{3,2,1,0} parameter(0)
+    %constant.1 = f32[] constant(3)
+    %broadcast.1 = f32[4,4,1024,1024]{3,2,1,0} broadcast(%constant.1), dimensions={}
+    ROOT %multiply.1 = f32[4,4,1024,1024]{3,2,1,0} multiply(%Arg_0.15, %broadcast.1)
+  }
+  )";
+  TF_ASSERT_OK_AND_ASSIGN(auto module,
+                          xla::ParseAndReturnUnverifiedModule(hlo));
+  auto comp = module->GetComputationWithName("fwd");
+  EXPECT_TRUE(comp != nullptr);
```

Comment:
Can you make this an ASSERT_NE? Assert because there is no point in continuing if this is a nullptr and _NE to get better error message if it fails.